PrismSampleApp/PrismSampleApp/ViewModels/MainPageViewModel.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Prism.AppModel;
+using Prism.Commands;
+using Prism.Navigation;
+using Prism.Services;
+using PrismSampleApp.Model;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+using PrismSampleApp.Resx;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Helpers;
+namespace PrismSampleApp.ViewModels
+{
+    public class MainPageViewModel : ViewModelBase , IPageLifecycleAware
+    {
+        private readonly INavigationService _navigationService;
+        private readonly IPageDialogService _pageDialogService;
+        public ICommand GoToApiContactsPageCommand { get; set; }
+        public ICommand GoToHomePageCommand { get; set; }
+        public ICommand ChangeLanguageCommand { get; set; }
+        private string _title;
+        public string Title
+        {
```

Comment:
I need all this code to be formatted throughout the app.

Reply:
all code has be formatted properly